### ISSUE TITLE
purging messages. 

### ISSUE THREAD

**Issue by ThiccestZexie** [AlexandreFPGoncalves/Eris#4]
An optional purge that saves the purged messages in a thread or a specific channel could be useful.

So for example. If I wanna purge a conversation but keep the logs I can run a command to remove them from channel 1 and keep them in channel 2. 

Usages: Banning someone and you don't want the message left on a channel but kept for later usages in a thread that gets deleted or something similar.

**Comment by AlexandreFPGoncalves** [comment]
I don't think it is possible for the purge event to capture what messages were deleted.
however, it is possible to maybe filter the last 10 messages, save them, and delete them afterwards.

But, do you really feel like it's something that would be used? Generally speaking, If I am purging messages I am purging trash that I don't want to see, and holding that information seems like a useless command.

**Comment by AlexandreFPGoncalves** [comment]
If it exists a client.on("messageDeleteBulk", ...) there must be a way to capture the messages and display them 😊 Explore and have fun!

![image](https://github.com/AlexandreFPGoncalves/Eris/assets/88319848/6c22ffe8-4456-4a1d-bf42-e49af893a25b)
